leave_manager/common/leave_manager.py

```diff
@@ -434,6 +460,11 @@ def reject_compensationLeave_request(request, leave_id):
         leave.save()
         
         if send_email_notification(update_details=update_details):
+            try:
+                leave_issuer_fcm = user.fcm_token
+                fcm(leave_issuer_fcm,request.user.get_full_name(),"reject_compensation")
+            except Exception as e:
+                print(e)

```

Comment:
return what on Exception? @AwaleRohin

Reply:
nothing because if there is fcm token then notification will be sent otherwise do nothing and cotinue with process